src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -47,6 +61,9 @@ export function RegisterForm() {
     if (values.name === undefined || values.name.length < minLength) {
       errors.name = t('registerForm.firstNameEmpty');
     }
+    if (values.roomNumber === undefined) {
+      errors.roomNumber = t('registerForm.roomNumberEmpty');
+    }
```

Comment:
If you add validation to `roomNumber` please validate vs correct room number 0+. Now someone can say my room is `dupa`